packages/excalidraw/element/cropElement.ts

```diff
@@ -138,6 +141,14 @@ export const cropElement = (
     nextHeight,
   );
 
+  // reset crop to null if we're back to orig size
+  if (
+    isCloseTo(crop.width, crop.naturalWidth) &&
+    isCloseTo(crop.height, crop.naturalHeight)
+  ) {
+    crop = null;
+  }
```

Comment:
@ryan-di resetting `element.crop` to null if we're back to natural dimensions ratio, unless there was some reason no to.

Reply:
Yup, that's good 👍